core/custom_filters/custom_filter_wizard_storage.py

```diff
@@ -0,0 +1,118 @@
+
+from core.custom_filters.custom_filter_registry_point import CustomFilterRegistryPoint
+from core.custom_filters import CustomFilterWizardInterface
+
+
+logger = logging.getLogger(__name__)
+
+
+class CustomFilterWizardStorage:
+    """
+        Class responsible for storing information on how to create filters based on the context of
+        a particular wizard creator. This hub provides information about the method for building
+        filters for a chosen field in specific object.
+
+        Constants:
+            __KEY_FOR_OBTAINING_CLASS: Indicates which element from the registry points should be
+                                       considered to retrieve information about the registered CustomFilterWizard class.
+            __FIELD: Identifies the first value of the final output tuple for building filters.
+            __FILTER: Identifies the second value of the final output tuple for building filters.
+            __VALUE: Identifies the third value of the final output tuple for building filters.
+    """
+
+    __KEY_FOR_OBTAINING_CLASS = 'class_reference'
+    __FIELD = 'field'
+    __FILTER = 'filter'
+    __VALUE = 'value'
+
+    @classmethod
+    def build_output_how_to_build_filter(cls, module_name: str, object_type: str, **kwargs) -> List[namedtuple]:
+        """
+            Build the final outcome for creating a filter based on the information provided by
+            a registered custom filter wizard for the specified module name and object type.
+
+            The output is a list of named tuples (from the collections package). Each named tuple is built
+            in the following format: <Type>(field=<str>, filter=<str>, value=<str>)
+
+            Example named tuple: BenefitPlan(field='income', filter='lt, gte, icontains, exact', value='')
+
+            Args:
+                module_name (str): The name of the module installed in openIMIS, required to retrieve information
+                                   about the possible ways of building filters for that specific module.
+                object_type (str): The name of the object type for which the filter building information is needed.
+
+            Returns:
+                List[namedtuple]: A list of named tuples representing the final outcome for creating a filter.
+        """
+        output_of_possible_filters = []
+        registered_filter_wizards = CustomFilterRegistryPoint.REGISTERED_CUSTOM_FILTER_WIZARDS
+        if module_name in registered_filter_wizards:
+            for registered_filter_wizard in registered_filter_wizards[module_name]:
+                if cls.__KEY_FOR_OBTAINING_CLASS in registered_filter_wizard:
+                    wizard_filter_class = cls.__create_instance_of_wizard_class(registered_filter_wizard)
+                    if cls.__check_object_type(wizard_filter_class, object_type):
+                        cls.__run_load_definition_object_in_wizard(wizard_filter_class, output_of_possible_filters, **kwargs)
```

Comment:
```suggestion
            for registered_filter_wizard in registered_filter_wizards[module_name]:
                if not cls.__KEY_FOR_OBTAINING_CLASS in registered_filter_wizard:
                    continue
                wizard_filter_class = cls.__create_instance_of_wizard_class(registered_filter_wizard)
                if not cls.__check_object_type(wizard_filter_class, object_type):
                    continue
                 cls.__run_load_definition_object_in_wizard(wizard_filter_class, output_of_possible_filters, **kwargs)
```
You can revert the ifs to avoid nesting